src/modules/container.rs

```diff
@@ -26,6 +37,11 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
             return Some("OCI".into());
         }
 
+        if check_if_docker_in_cgroup().unwrap() {
```

Comment:
Have you tested this? This looks like it will crash if `/proc/1/cgroup` does not exist. Maybe just ignore the error and use the following:
```suggestion
        if check_if_docker_in_cgroup().unwrap_or(false) {
```